admin_pages/registration_form/templates/questions_main_meta_box.template.php

```diff
@@ -310,27 +310,27 @@ class="option-desc regular-text"
                                             <td class="option-value-cell">
                                                 <input type="hidden"
                                                        class="QSO_order"
-                                                       name="question_options[<?php echo $count; ?>][QSO_order]"
-                                                       value="<?php echo $count; // escape not needed ?>"
+                                                       name="question_options[<?php echo esc_attr($count); ?>][QSO_order]"
+                                                       value="<?php echo esc_attr($count); ?>"
```

Comment:
hmmmph...  

i mean... i'd prefer to NOT have to do anything to this since it is clearly set as an integer and incremented using `++` so this var could not possibly be something else... that said, and again, wp.org is not being reasonable about that kind of logic so it might be better to show that this is being handled (even if not needed).

however... `esc_attr()` runs filters and a bunch of other logic which is totally overkill, so let's just use `absint()` for known integer values because it will be a lot quicker. I realize that the difference is pretty small, but on EventSmart where we have thousands of active sites running, these miniscule differences can add up to a lot of extra CPU time over a year.